mscore/plugin/pluginCreator.ui

```diff
@@ -46,7 +46,7 @@
           </sizepolicy>
          </property>
          <property name="text">
-          <string>Run</string>
+          <string>&amp;Run</string>
```

Comment:
While a good idea in general, it'd render all translation invalid, and without a proper tranbslation infrastructure in place

Reply:
Ah, makes sense. Well, at least the idea is here for anyone concerned